pkg/provider/provider_helpers.go

```diff
@@ -3,20 +3,17 @@ package provider
 import (
 	"crypto/rsa"
 	"encoding/json"
-	"errors"
 	"fmt"
 	"io"
 	"net/http"
 	"net/url"
-	"os"
 	"strconv"
 	"strings"
 	"time"
 
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/provider"
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/mitchellh/go-homedir"
```

Comment:
nit: I see this only lives in framework PoC, maybe let's remove it from there too and get rid of this dependency at all?

Reply:
Done.